### PR TITLE
add: script to bootstrap a devel server

### PR DESCRIPTION
**Pull request by pi-rho**
A idempotent script to bootstrap development server. This script, when run in an empty-ish directory will clone moth into ./moth and create a directory at ./puzzles. The puzzle directory is communicated with the user and the development server starts on port 8080.

### CODE REVIEW

File: devel.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+#
+# Script to clone and start a development server
+
+if [[ -f tools/devel-server.py ]]; then
+	cat <<EOM
+This script is intended to be used to bootstrap a moth development server. It
+looks like you're running the script from a moth repository working directory.
+
+    $ mkdir /tmp/moth
+	$ cd /tmp/moth
```
Comment (nealey) [inline devel.sh]: TABS STRIKE AGAIN

File: devel.sh [hunk 2]
```diff
@@ -0,0 +1,25 @@
+#!/bin/bash -e
```
Comment (nealey) [inline devel.sh]: bash isn't necessary

File: devel.sh [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+#
+# Script to clone and start a development server
+
+if [[ -f tools/devel-server.py ]]; then
```
Comment (nealey) [inline devel.sh]: change to `[` and `]` to not need bash

File: devel.sh [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+    $ mkdir /tmp/moth
+	$ cd /tmp/moth
+	$ curl https://raw.githubusercontent.com/dirtbags/moth/master/devel.sh | bash
+EOM
+	exit 1
+fi
+
+[[ -d puzzles  ]] || mkdir -p puzzles
+[[ -d moth/bin ]] || git clone https://github.com/dirtbags/moth.git
```
Comment (nealey) [inline devel.sh]: same `[` and `]` here

File: devel.sh (hunk 1 above)
Comment (pi-rho) [reply devel.sh]: fixed!

**Comment by pi-rho** [comment]
I'm sad that it and the docker cruft exists. Both are a good demonstration that the development server is super easy to invoke. Like bash being unnecessary, docker is unnecessary.  The only benefit I saw in using docker is that you could invoke it's magic to go pull down a clone of the moth repository and start up the server. This script provides the same magic without depending on docker, if you're the type of person that will `curl http:// ... | sh`.

**Pull request merged**